tools/build-on-openbsd

```diff
@@ -16,12 +22,23 @@ pkgs="
    py3-setuptools
    py3-yaml
    sudo--
+   wget
```

Comment:
What's that used for?

Reply:
I don't recall right off, so I will take out and re-run to see if I still get an error in the logs about this.  This might have been an issue when I was doing a build on older version of cloud-init some how and after I cleaned all the cached stuff, this might not be an issue anymore.

Reply:
okay, so that's CloudStack specific.